docker-entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+yarn build
+export BACKEND_URL=$URL
```

Comment:
Where does $URL come from?

Reply:
you post it in docker command for example "-e URL=https://demo.spreecommerce.org"

Reply:
Can we keep the original BACKEND_URL env variable? It will then be consistent with the docs that we published for VSF